### PR TITLE
Add Wormhole Chain IDs and license comments atop files

### PR DESCRIPTION
**Pull request by nik-suri**

### CODE REVIEW

File: src/Chains.sol [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+uint16 constant CHAIN_ID_NEAR = 15;
+uint16 constant CHAIN_ID_MOONBEAM = 16;
+uint16 constant CHAIN_ID_NEON = 17;
+uint16 constant CHAIN_ID_TERRA2 = 18;
+uint16 constant CHAIN_ID_INJECTIVE = 19;
+uint16 constant CHAIN_ID_OSMOSIS = 20;
+uint16 constant CHAIN_ID_SUI = 21;
+uint16 constant CHAIN_ID_APTOS = 22;
+uint16 constant CHAIN_ID_ARBITRUM = 23;
+uint16 constant CHAIN_ID_OPTIMISM = 24;
```
Comment (barnjamin) [inline src/Chains.sol:29]: gnosis?

File: src/Chains.sol [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+uint16 constant CHAIN_ID_PYTHNET = 26;
+uint16 constant CHAIN_ID_XPLA = 28;
+uint16 constant CHAIN_ID_BTC = 29;
+uint16 constant CHAIN_ID_BASE = 30;
+uint16 constant CHAIN_ID_SEI = 32;
```
Comment (barnjamin) [inline src/Chains.sol:34]: rootstock = 33?

File: src/Chains.sol (hunk 1 above)
Comment (nik-suri) [reply src/Chains.sol:29]: I copy-pasted from the go sdk `vaa/structs.go`, so it must be missing there too.

Will add here and there as well.

File: src/Chains.sol (hunk 2 above)
Comment (nik-suri) [reply src/Chains.sol:34]: Same comment as above.

File: src/Chains.sol [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+uint16 constant CHAIN_ID_SCROLL = 34;
+uint16 constant CHAIN_ID_MANTLE = 35;
+uint16 constant CHAIN_ID_WORMCHAIN = 3104;
+uint16 constant CHAIN_ID_COSMOSHUB = 4000;
+uint16 constant CHAIN_ID_EVMOS = 4001;
+uint16 constant CHAIN_ID_KUJIRA = 4002;
+uint16 constant CHAIN_ID_NEUTRON = 4003;
+uint16 constant CHAIN_ID_CELESTIA = 4004;
+uint16 constant CHAIN_ID_SEPOLIA = 10002;
```
Comment (barnjamin) [inline src/Chains.sol:43]: new sepolias?
```
  arbitrum_sepolia: 10003,
  base_sepolia: 10004,
  optimism_sepolia: 10005,
```

Comment (nik-suri) [reply src/Chains.sol:45]: Will add here and to Go sdk as well.

**Review by barnjamin — APPROVED** [review]

File: src/Chains.sol [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+uint16 constant CHAIN_ID_TERRA2 = 18;
+uint16 constant CHAIN_ID_INJECTIVE = 19;
+uint16 constant CHAIN_ID_OSMOSIS = 20;
+uint16 constant CHAIN_ID_SUI = 21;
+uint16 constant CHAIN_ID_APTOS = 22;
+uint16 constant CHAIN_ID_ARBITRUM = 23;
+uint16 constant CHAIN_ID_OPTIMISM = 24;
+uint16 constant CHAIN_ID_GNOSIS = 25;
+uint16 constant CHAIN_ID_PYTHNET = 26;
+uint16 constant CHAIN_ID_XPLA = 28;
```
Comment (SEJeff) [inline src/Chains.sol:32]: Is there a chain id for 27?

File: src/Chains.sol [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+uint16 constant CHAIN_ID_BTC = 29;
+uint16 constant CHAIN_ID_BASE = 30;
```
Comment (SEJeff) [inline src/Chains.sol:34]: Ditto for 31.

Comment (barnjamin) [reply src/Chains.sol:34]: https://github.com/wormhole-foundation/wormhole/blob/main/sdk/js/src/utils/consts.ts this is what I've been using as the source of truth

File: src/Chains.sol (hunk 4 above)
Comment (nik-suri) [reply src/Chains.sol:32]: I do not believe that there is.

File: src/Chains.sol (hunk 5 above)
Comment (nik-suri) [reply src/Chains.sol:34]: Same ^ I do not believe that there is a chain id for 31.

**Review by chase-45 — APPROVED** [review]

**Pull request merged**